### PR TITLE
43 set up tailwind css for vue

### LINKED ISSUE #43: Set up TailwindCSS for Vue
### Description
Set up Tailwind for Vue to manage styling easier. If we have the time to do so with the current priorities, rewrite the Vue files currently using `style scoped` to use Tailwind. If not then let the team know and we'll leave that for a bit later. 

### To-do
* [ ] Set up Tailwind
* [ ] *Rewrite current CSS code into Tailwind

### PR DESCRIPTION
**Pull request by keyman015**
## Description

TailwindCSS has been setup and some samples have been completed for the NavBar, NavBarItem and Icons. When adding custom definitions such as colours or other settings, they must be added to `tailwind.config.js`. Now, there should be little to no blocked css in the scoped tags as styling should use the classes now. 

To support different screen sizes you must follow the convention of placing the global style first then the screen sizes above it, for instance:
```
class="hidden md:flex"
```
This will hide the component for screens BELOW medium and have `display:flex` for screens of medium and above. To get further support or other class names please use tailwind's quick search (https://tailwindcss.com/).

## Notes
To support custom class passing to components, a new prop must be added to components. This can be seen in the NavBarItem component.

## Misc
Closes #43

### CODE REVIEW

File: src/client/src/App.vue [hunk 1]
```diff
@@ -6,19 +6,19 @@ import NavBar from './components/NavBar.vue'
...
 
-    <main class="main-content">
+    <main id="main-content" class="bg-background pt-8">
       <RouterView />
     </main>
   </div>
 </template>
 
-<style scoped>
+<!-- <style scoped>
```
Comment (paulmmaingi) [inline src/client/src/App.vue:21]: Should we leave the commented previous styles?

**Review by paulmmaingi — APPROVED** [review]
LGTM.
A few styles did get lost in the translation to Tailwind e.g. the navbar height but these should be fixable when we do the final styling run and decide on the final styles for each component.

File: src/client/src/App.vue (hunk 1 above)
Comment (sellsol) [reply src/client/src/App.vue:21]: Previous styles should just be removed if they've been replaced

Comment (keyman015) [reply src/client/src/App.vue]: Yes, removed now.

File: src/client/src/components/NavBarItem.vue [hunk 2]
```diff
@@ -45,28 +53,37 @@ export default {
 
 <template>
   <!-- If it's an external link -->
```
[the remark this answers is not in the document]
Comment (keyman015) [reply src/client/src/components/NavBarItem.vue]: Fixed

**Review by sellsol — APPROVED** [review]

**Pull request merged**